accounts/migrations/0001_initial.py

```diff
@@ -8,17 +8,45 @@
 class Migration(SchemaMigration):
 
     def forwards(self, orm):
+        # Adding model 'TwitterInfo'
```

Comment:
I don't think migrations are "just a record".  I think if someone is 
behind in the code history and does a pull, the new migrations get run 
to bring their database in line with the current.  Similarly if someone 
checks out a whole new copy, I think the migration files are the way the 
db gets set up?

On 01/22/2016 02:37 PM, Amy Zhang wrote:
>
> In accounts/migrations/0001_initial.py 
> <https://github.com/haystack/eyebrowse-server/pull/141#discussion_r50580575>:
>
> > @@ -8,17 +8,45 @@
> >  class Migration(SchemaMigration):
> >
> >      def forwards(self, orm):
> > +        # Adding model 'TwitterInfo'
>
> Can you just revert all your changes to 0001_initial.py for the sake 
> of this pull request? I don't see why we need these changes. After all 
> migration files are just to keep a record of how the db has been 
> migrated over a time. No one should be running these files going forward.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/haystack/eyebrowse-server/pull/141/files#r50580575>.
>